lpcompiler.pas

```diff
@@ -1310,7 +1324,10 @@ function TLapeCompiler.ParseMethodHeader(out Name: lpString; addToScope: Boolean
         end;
       until (Tokenizer.Tok = tk_sym_ParenthesisClose);
 
-    if isFunction then
+    if Result.isOperator and (Result.Params.Count <> 2) then
```

Comment:
I guess it can be done. And that would solve some of the hacky evaluating I've done ;P